app/models/individual_market/immigration_information.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module IndividualMarket
+  # This class represents immigration information for an applicant.
+  # This is replacement for VlpDocument model in the main application.
+  class ImmigrationInformation
+    include Mongoid::Document
+    include Mongoid::Timestamps
+
+    # @!attribute applicant
+    #   @return [IndividualMarket::Applicant] The applicant to which this immigration information belongs to
+    embedded_in :applicant, class_name: 'IndividualMarket::Applicant'
+
+    # Enable the commented fields below or add new fields (more descriptive) as needed.
```

Comment:
Glad to see this commented out, as it probably makes sense to reconsider the way we have this implemented in the legacy models.  Dumping every field into a single model makes distinguishing which type of document has which relevant fields a real pain.  We could use a more thoughtful approach here.